parts/k8s/cloud-init/artifacts/cse_customcloud.sh

```diff
@@ -58,6 +58,7 @@ configureK8sCustomCloud() {
         KUBERNETES_FILE_DIR=$(dirname "${AZURE_JSON_PATH}")
         K8S_CLIENT_CERT_PATH="${KUBERNETES_FILE_DIR}/k8s_auth_certificate.pfx"
         echo $SERVICE_PRINCIPAL_CLIENT_SECRET_CERT | base64 --decode > $K8S_CLIENT_CERT_PATH
+        # shellcheck disable=SC2002
```

Comment:
This usage and the one below are piping `cat` in order to modify a single file in-place. I don't know of another way to do that with `jq` except by using a temp file.